### PR TITLE
Use default virtual network switch

### PR DESCRIPTION
**Pull request by Wiezzel**

### CODE REVIEW

File: golem/docker/hypervisor/hyperv.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import logging
+import os
```
Comment (maaktweluit) [inline golem/docker/hypervisor/hyperv.py:2]: now it `import os` and `from os import path`, maybe those 2 lines can be joined to make it cleaner?

File: golem/docker/hypervisor/hyperv.py [hunk 2]
```diff
@@ -114,22 +110,19 @@ def update_work_dir(self, work_dir: Path) -> None:
...
-        Create the virtual switch required to start a hyperv machine.
+        Get name of the host machine which could be used for sharing
+        directories with Hyper-V VMs connected to Golem's virtual switch.
         """
-        return cls._run_ps(cls.SETUP_SWITCH_PATH, 20)
+        return os.getenv('COMPUTERNAME')
 
     @classmethod
-    def _run_ps(cls, script, timeout=10):
+    def _run_ps(cls, script, timeout=SCRIPT_TIMEOUT):
```
Comment (maaktweluit) [inline golem/docker/hypervisor/hyperv.py:127]: nice one! missed that in my PR!

File: tests/golem/docker/test_hyperv.py [hunk 3]
```diff
@@ -27,43 +27,22 @@ def _assert_param(self, args, name, value):
...
-            ],
-            timeout=20,
-            check=True,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-        )
-
-    @patch(PATCH_BASE + '.subprocess.run',
-           side_effect=subprocess.CalledProcessError(1, 'a', stderr=b''))
-    def test_parse_create_vnet_switch_fail(self, _):
```
Comment (maaktweluit) [inline tests/golem/docker/test_hyperv.py:63]: maybe keep the failure case for better coverage?

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

Some small comments, none are breaking. Will approve on reply

File: golem/docker/hypervisor/hyperv.py (hunk 1 above)
Comment (Wiezzel) [reply golem/docker/hypervisor/hyperv.py:2]: Done.

File: tests/golem/docker/test_hyperv.py (hunk 3 above)
Comment (Wiezzel) [reply tests/golem/docker/test_hyperv.py:63]: Done.

**Review by maaktweluit — APPROVED** [review]

**Pull request merged**